inventory.example

```diff
@@ -42,6 +42,10 @@ dcos_legacy_node_type_name=slave_public
 agents_private
 agents_public
 
+[win_agents]
```

Comment:
rename to `agents_windows` to match the same naming as other agents

Reply:
corrected